project/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
+ENTRYPOINT ["python3"]
+CMD ["api.py"]
```

Comment:
Why do we need both `CMD` and `ENTRYPOINT` here and not just use `ENTRYPOINT ["python3", "api.py"]` here?